include/deal.II/grid/grid_tools.h

```diff
@@ -170,6 +171,12 @@ namespace GridTools
   template <int dim, typename T>
   double cell_measure (const T &, ...);
 
+  /**
+   * Compute the smallest box containing the entire triangulation.
+   */
+  template <int dim, int spacedim>
+  BoundingBox<spacedim> compute_bounding_box(const Triangulation<dim, spacedim> &triangulation);
```

Comment:
FYI (maybe helps with implementation, or re-use it) https://www.dealii.org/developer/doxygen/deal.II/namespaceGridTools.html#a0374dd9eae73c3c2badaa479e8ba8d55

Reply:
Oh, good memory, @davydden -- we already have this function if you just pass `[](...) { return true; }` to that function!

Reply:
> Oh, good memory

that's because my colleague @vishalkenchan added it in https://github.com/dealii/dealii/commit/0cc970cc57bb5a1ce9ee031d582e66f61a4c9fad :smile:

Reply:
Oof how did I miss that?!